OpenRA.Mods.Common/UtilityCommands/ExtractZeroBraneStudioLuaAPI.cs

```diff
@@ -0,0 +1,126 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Linq;
+using System.Reflection;
+using OpenRA.Scripting;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.UtilityCommands
+{
+	// See https://studio.zerobrane.com/doc-api-auto-complete for reference
+	class ExtractZeroBraneStudioLuaAPI : IUtilityCommand
+	{
+		public string Name { get { return "--zbstudio-lua-api"; } }
+
+		public bool ValidateArguments(string[] args)
+		{
+			return true;
+		}
+
+		[Desc("Generate a ZeroBrane Studio Lua API and auto-complete description.")]
```

Comment:
The IDE has no package management https://github.com/pkulchenko/ZeroBraneStudio/issues/528 so it can't be properly automatized. I don't want to maintain this at https://github.com/pkulchenko/ZeroBranePackage and a separate `zerobrane-completion.zip`. Also I suggest not to over-engineer a release management for such a small addon only a very small percentage of our very advanced users will use. I left documentation at https://github.com/pkulchenko/ZeroBranePackage/blob/master/openra.lua#L9 to make it easy for everyone to update it.